lib/gnu/md5.c

```diff
@@ -22,7 +22,7 @@
 #include <config.h>
 
 #if HAVE_OPENSSL_MD5
-# define GL_OPENSSL_INLINE _GL_EXTERN_INLINE
+# define GL_OPENSSL_INLINE _GL_INLINE
```

Comment:
I'm guessing what's happening is that gnulib assumes that if openssl headers are present, then we must be linking against it. So this should work for our purposes, unless we ever do really want to link against openssl.

Reply:
If you do ./configure --with-openssl, aren't you asking for linking against openssl?  So yeah, that's a pretty safe assumption on gnulib's part.  I hate to even suggest this because it means spending more time with gnulib, but maybe it would be worth looking at what would be required for us to actually links against openssl in this case.

Reply:
We don't use openssl and don't want the --with-openssl option, but gnulib's gl_* macros add it unconditionally, which is annoying. However gnulib appears to assume that the md5 function is extern if the headers are simply installed on the local machine, regardless of --with-openssl.